rdtools/normalization.py

```diff
@@ -14,20 +14,22 @@ class ConvergenceError(Exception):
 def normalize_with_expected_power(pv, power_expected, poa_global,
                                   pv_input='power'):
     '''
-    Normalize pv output based on expected PV power.
+    Normalize PV power or energy based on expected PV power.
 
     Parameters
     ----------
     pv : pd.Series
         Right-labeled time series PV energy or power. If energy, should *not*
-        be cumulative, but only for preceding time step.
+        be cumulative, but only for preceding time step. Type (energy or power)
+        must be specified in the pv_input parameter.
     power_expected : pd.Series
-        Right-labeled time series of expected PV power.
+        Right-labeled time series of expected PV power. (Note: Expected energy
+        is not supported.)
     poa_global : pd.Series
         Right-labeled time series of plane-of-array irradiance associated with
         `expected_power`
-    pv_input : str
-        'power' or 'energy' to specify type of input used for pv parameter
+    pv_input : {'power' or 'energy'}
+        Specifies the type of input used for pv parameter. Default: 'power'
```

Comment:
```suggestion
    pv_input : {'power' or 'energy'}, default 'power'
        Specifies the type of input used for `pv` parameter.
```

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html Section 4 seems to indicate that the default should be in the description. But maybe I'm misinterpreting?

Reply:
Hmm I thought `paramname : type, default X` was fairly standard but I'm not finding many examples to support that in the big packages.  Only one I found so far is the `axis` parameter here, but it's not really apples to apples: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.shift.html

Okay with me to follow the numpydoc guidelines.  I do still think `pv` needs the backticks.